### PR TITLE
Add bit rate

### PR DESCRIPTION
**Pull request by roideuniverse**
Extract the audio bit rate from video file and use that bitrate in the shell command to trim the length of the video.

### CODE REVIEW

File: src/main/java/org/ffmpeg/android/FfmpegController.java [hunk 1]
```diff
@@ -602,33 +604,89 @@ public void squareCrop(File inputFile, File outputFile, int squareSize, ShellCal
...
-        c.add("libx264");
-        c.add("-acodec");
-
-        //c.add("-c");
-        c.add("copy");
-
-        c.add(outputFile.getAbsolutePath());
-        execFFMPEG(c, sc);
-    }
+	public void trim(final File inputFile, final File outputFile, final ShellCallback sc) throws IOException, InterruptedException
```
Comment (BrandonRomano) [inline src/main/java/org/ffmpeg/android/FfmpegController.java:732]: This is scary, but it works, and we can bank of it working consistently -- I just still want to test this on a few different devices.

**Comment by roideuniverse** [comment]
@BrandonRomano : So doing both the process of squaring and rotating the video was not working, so I had to split into two.
First - I create the square video and save it in temp file.
Second - Rotate the square video and delete the temp file.

File: src/main/java/org/ffmpeg/android/FfmpegController.java [hunk 2]
```diff
@@ -560,75 +564,301 @@ public Clip combineAudioAndVideo (Clip videoIn, Clip audioIn, Clip out, ShellCal
...
+		c.add(tempFile.getAbsolutePath());
+
+		StringBuilder command = new StringBuilder();
+		for(String s : c)
+		{
+			command.append(s);
+			command.append(" ");
+		}
+
+		Log.d("", "Brandon: Command:" + command.toString());
```
Comment (BrandonRomano) [inline src/main/java/org/ffmpeg/android/FfmpegController.java]: We can get rid of this now ( or at least remove the `Brandon:` part)

**Comment by BrandonRomano** [comment]
Okay, this is really awesome :clap:.  Super great job, I owe you lunch now :stuck_out_tongue_winking_eye: 

:+1: :+1:

**Comment by roideuniverse** [comment]
Lol thanks :smile_cat:  - merging

**Pull request merged**